README.md

```diff
@@ -20,6 +20,32 @@ sequenceDiagram
 ```
 
 
+# Objectives and Explanation of OAuth Status Assertions
+
+Status assertions are used to provide time sensitive claims that impact the validity of digital credentials without the need for a direct request from third parties. They are periodically issued to the holders of the credentials, who can present them to the verifiers together with the corresponding digital credentials.
+
+## Key aspects of the concept of OAuth Status Assertions:
+OAuth Status Assertions provide a robust method of verifying the validity of digital credentials without the need for verifiers to access external sources or third parties.
```

Comment:
Perhaps we should stick to the terminology: "verify the status of the digital credentials". It is up to the Verifier to determine what valid means in their context.